geopandas/sindex.py

```diff
@@ -14,14 +15,7 @@ def _get_sindex_class():
     Required to comply with _compat.USE_PYGEOS.
     The selection order goes PyGEOS > RTree > Error.
     """
-    if compat.USE_SHAPELY_20 or compat.USE_PYGEOS:
-        return PyGEOSSTRTreeIndex
-    if compat.HAS_RTREE:
-        return RTreeIndex
-    raise ImportError(
-        "Spatial indexes require either `rtree` or `pygeos`. "
-        "See installation instructions at https://geopandas.org/install.html"
-    )
+    return PyGEOSSTRTreeIndex
```

Comment:
This commit "sindex partial" (https://github.com/geopandas/geopandas/pull/3013/commits/116a42fd37f6166d15002975d7e8cec6645f35f3), I assume you mostly just dedented the code? (it's quite impossible to check the diff here ;))

Reply:
Yes, removed rtree index and dedented pygeos index.